src/plugins/timeline/TimelineViewLayout.vue

```diff
@@ -86,15 +95,24 @@ export default {
         return {
             items: [],
             timeSystems: [],
-            height: 0
+            height: 0,
+            useIndependentTime: this.domainObject.configuration ? this.domainObject.configuration.useIndependentTime : false,
```

Comment:
looks like `initialize` creates empty object so `this.domainObject.configuration` should be `{}` by default.
so I think can directly use `this.domainObject.configuration.useIndependentTime`

Reply:
Good point. I'll have to write an interceptor for timestrip views that don't have the configuration.

Reply:
Added an interceptor.